src/lib/datepicker/datepicker.md

```diff
@@ -228,3 +228,52 @@ application root module.
 })
 export class MyApp {}
 ```
+### Accessibility
+The `MdDatepickerInput` directive adds `aria-haspopup` attribute to the native input element, and it
+triggers a calendar dialog with `role="dialog"`.
+
+`MdDatepickerIntl` includes strings that are used for `aria-label`s. The datepicker input
+should have a placeholder or be given a meaningful label via `aria-label`, `aria-labelledby` or
+`MdDatepickerIntl`.
+
+#### Keyboard shortcuts
+The keyboard shortcuts to handle datepicker are:
+
+| Shortcut             | Action                            |
+|----------------------|-----------------------------------|
+| `ALT` + `DOWN_ARROW` | Open the calendar pop-up          |
+| `ESCAPE`             | Close the calendar pop-up         |
+
+
+In month view:
+
+| Shortcut             | Action                            |
+|----------------------|-----------------------------------|
+| `LEFT_ARROW`         | Goes to previous day              |
+| `RIGHT_ARROW`        | Goes to next day                  |
+| `UP_ARROW`           | Goes to previous 7 days           |
+| `DOWN_ARROW`         | Goes to next 7 days               |
```

Comment:
`Goes` -> `Go` for each of these

For up and down arrow, I'd say `Go to same day in the next week`